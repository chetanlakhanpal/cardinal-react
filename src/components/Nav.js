import React from 'react'
import { Link } from 'react-router-dom'
import styled from 'styled-components'

const Navbar = styled.nav`
background-color: rgba(0, 0, 0, .85);
-webkit-backdrop-filter: saturate(180%) blur(20px);
backdrop-filter: saturate(180%) blur(20px);
`
const StyledLink = styled(Link)`
font-size: 20px;
color: #999;
transition: ease-in-out color .15s;
&:hover {
  color: #fff;
  text-decoration: none;
}
`

const Nav = () => {
    return (
      <Navbar className="py-1">
        <div className="container d-flex flex-column flex-md-row justify-content-between">
          <StyledLink to="/Dashboard">Dashboard</StyledLink>
        </div>
      </Navbar>)
    }

export default Nav