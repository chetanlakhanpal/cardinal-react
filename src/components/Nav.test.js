import React from 'react'
import { shallow } from 'enzyme'
import Nav from './Nav'

describe('Nav Component', () => {
  const navComponent = shallow(<Nav/>)

  it('renders component', () => {
    expect(navComponent).toMatchSnapshot()
  })
})



