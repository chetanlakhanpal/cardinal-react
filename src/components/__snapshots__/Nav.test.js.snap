// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav Component renders component 1`] = `ShallowWrapper {}`;
